docker-entrypoints/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --argstr type test -i bash /app/shell.nix
+
+# This entrypoint is generally used by the CI/CD pipeline.
+# This runs the given commands in an environment that has all the dependencies
+# from `run` and some additional dependencies to run the tests.
+# e.g. geckodriver for browser tests.
+
+poetry run "$@"
```

Comment:
Why do we need three files with exactly the same content?

Reply:
#!nix-shell --argstr type **test** -i bash /app/shell.nix

Reply:
Here we specify the `type test`